lib/createEmployee.ts

```diff
@@ -0,0 +1,38 @@
+"use server";
+import { cookies } from "next/headers";
+
+export const createEmployee = async (employeeData: Employee) => {
+  try {
+    const cookieStore = cookies();
+    const accessToken = cookieStore.get("token")?.value;
+
+    if (!accessToken) {
+      throw new Error("No access token found.");
+    }
+
+    console.log("Access Token:", accessToken);
+    console.log("Employee Data:", employeeData);
+
+    const res = await fetch(
+      "https://employee-management-api-xj3a.onrender.com/employees",
```

Comment:
You can create a utils or config folder where you'll store the api url in a file.
Like this:

export const config = {
  BASE_API_URL: "https://employee-management-api-xj3a.onrender.com",
};

In this file you can do something like this:
await fetch(`${config.BASE_API_URL}/employees`)

Reply:
Thank you, I actually missed that.